django/db/models/fields/json.py

```diff
@@ -313,7 +313,11 @@ def process_rhs(self, compiler, connection):
     def as_oracle(self, compiler, connection):
         lhs, lhs_params = self.process_lhs(compiler, connection)
         rhs, rhs_params = self.process_rhs(compiler, connection)
-        return f"JSON_EQUAL({lhs}, {rhs})", (*lhs_params, *rhs_params)
+        db_version = connection.get_database_version()
+        if db_version[0] > 19:
```

Comment:
This is related with primitives support so we can use `supports_primitives_in_json_field`:
```suggestion
        if connection.features.supports_primitives_in_json_field:
```

Reply:
good to know, thanks. Done.